packages/esbuild-plugin/src/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+const babel = require('@babel/core');
+const stylexBabelPlugin = require('@stylexjs/babel-plugin');
+const flowSyntaxPlugin = require('@babel/plugin-syntax-flow');
+const typescriptSyntaxPlugin = require('@babel/plugin-syntax-typescript');
+const jsxSyntaxPlugin = require('@babel/plugin-syntax-jsx');
+const path = require('path');
+const fs = require('fs/promises');
+
+const PACKAGE_NAME = 'esbuild-plugin-stylex';
+
+const IS_DEV_ENV =
+  process.env.NODE_ENV === 'development' ||
+  process.env.BABEL_ENV === 'development';
+
+function stylexPlugin({
+  dev = IS_DEV_ENV,
+  unstable_moduleResolution = { type: 'commonJS', rootDir: process.cwd() },
+  stylexImports = ['@stylexjs/stylex'],
+  // absolute path to bundled CSS file
+  absoluteFilePath = path.resolve(__dirname, 'stylex.css'),
```

Comment:
Can we rename this option to something more descriptive?

Reply:
changed it to `generatedCSSFileName`. I don't think it should have the `absolute` in it's name as it can also consume relative file paths, though using absolute path is a recommendation.